tools/depositCollectionPubs.js

```diff
@@ -19,14 +18,36 @@ const main = async () => {
 	const collection = await Collection.findOne({
 		where: { communityId: community.id, slug: collectionSlug },
 	});
-	const collectionPubs = await CollectionPub.findAll({ where: { collectionId: collection.id } });
-	console.log(`=== Will deposit 1 Collection and ${collectionPubs.length} Pubs`);
+	const collectionId = collection.id;
+	const collectionPubs = await CollectionPub.findAll({
+		where: { collectionId },
+		include: [
+			{
+				model: Pub,
+				as: 'pub',
+				include: [
+					{
+						model: Release,
+						as: 'releases',
+						attributes: ['id'],
+					},
+				],
+			},
+		],
+	});
+	const releasedPubs = collectionPubs.reduce((acc, cp) => {
```

Comment:
Some naming-related pedantry — `releasedPubs` is still a list of `CollectionPub` objects, so perhaps:
- The name should reflect that, e.g. `collectionPubsWithReleases`, or
- The `reduce` should collect the Pubs: `acc.push(cp.pub)`